documentation/src/main/asciidoc/topics/con_xsite_expiration.adoc

```diff
@@ -1,7 +1,14 @@
 [id='xsite_expiration-{context}']
-= Expiration and Cross-Site Replication
-{brandname} expiration controls how long entries remain in the cache.
+= Expiration with Cross-Site Replication
+Expiration removes cache entries based on time.
+{brandname} provides two ways to configure expiration for entries:
 
-* `lifespan` expiration is suitable for cross-site replication. When entries reach the maximum lifespan, {brandname} expires them independently of the remote sites.
+`lifespan`:: Sets the maximum amount of time that entries can exist. +
+When you set `lifespan` with cross-site replication, {brandname} clusters expire entries independently of remote sites.
 
-* `max-idle` expiration does not work with cross-site replication. {brandname} cannot determine when cache entries reach the idle timeout in remote sites.
+`max-idle`:: Specifies how long entries can exist based on read or write operations in a given time period. +
+When you set a `max-idle` with cross-site replication, {brandname} clusters send touch commands to coordinate idle timeout values with remote sites. +
```

Comment:
This is a bit generalized, but maybe that is the point? Seems fine though.

Reply:
@wburns Yes it's a bit generalized but it seemed preferable to getting too low level. Do you think there are any points we should be more specific about or could we just merge this?